hw/ip/csrng/data/csrng_testplan.hjson

```diff
@@ -104,23 +130,46 @@
             Individual commands and command options that will be covered include:
             - app
             - acmd, clen, flags, glen
-            Crosses of app/acmd and acmd/clen, acmd/flags, acmd/glen
             - continuous/non-continuous valid
+            Crosses of
+            - app/acmd
+            - For the instantiate and reseed command:
+              - flag0 false and clen 0
+              - flag0 false and clen >0
+              - flag0 true and clen 0
+              - flag0 true and clen >0
+            - For the generate command:
+              - that each of the 12 bits in glen is set and unset at least once
             '''
     }
     {
-      name: err_test_cg
+      name: csrng_internal_cg
+      desc: '''
+            Covers that all csrng internal states have been hit:
+            - genbits has had each bit set and unset at least once
```

Comment:
As just discussed, likely unnecessary because it's an output that will have toggle coverage checked

Reply:
Removed this, thanks for the reminder.